mono/mini/debugger-agent.c

```diff
@@ -4507,6 +4696,73 @@ set_breakpoint (MonoMethod *method, long il_offset, EventRequest *req, MonoError
 	return bp;
 }
 
+#ifdef IL2CPP_MONO_DEBUGGER
+
+static MonoBreakpoint* set_breakpoint_fast(Il2CppSequencePointC *sp, EventRequest *req, MonoError *error)
+{
+	MonoBreakpoint *bp;
+	GHashTableIter iter, iter2;
+	MonoDomain *domain;
+	MonoMethod *m;
+	MonoSeqPointInfo *seq_points;
+	GPtrArray *methods;
+	GPtrArray *method_domains;
+	GPtrArray *method_seq_points;
+	int i;
+
+	if (error)
```

Comment:
Are we passing in errors to this function?
I think the convention is to do something like
MonoError err
mono_error_init(&err)
function_that_consumes_error(&err)

this function shouldn't need to do this if it's owners are following convention.

Reply:
I'm not entirely sure. This was brought over from the hackweek debugger. It seems to work, but we probably have not tested the error conditions. More generally though, we've stubbed out most of the Mono error handling code so far. I think we will need to revisit that if it becomes as issue later.